### PR TITLE
README Edits

### PR DESCRIPTION
**Pull request by Kodylow**
Learning about Sv2 this weekend, made some spelling fixes and simplified some of the explanations in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,90 @@
...
-   that wants to expand the protocol or implement a role (e.g. a pool wanting to support Sv2, a
-   mining-device/hashrate producer that wants to integrate Sv2 into their firmware, a Bitcoin node
-   that wants to implement Template Provider capabilities to build the `blocktemplate`).
+The goals of this project are to provide:
 
-1. The above Rust primitives as a C library, making it available to non-Rust users via a C
-   bindings.
+1. A robust set of Stratum V2 (Sv2) primitives as Rust library crates which anyone can use
+   to expand the protocol or implement a role. For example:
+   a) Pools supporting Sv2
```
Comment (ccdle12) [inline README.md:49]: When creating these sub-bullet points - `a) , b), c)` don't appear on new lines when the markdown is rendered

Comment (Kodylow) [reply README.md]: Fixed it, thanks!

**Comment by ccdle12** [comment]
Thanks for the contribution!, this looks good to me. Ping another reviewer for an ok: @Fi3 @rrybarczyk

File: README.md [hunk 2]
```diff
@@ -1,90 +1,91 @@
...
 
-1. A robust set of Stratum V2 (Sv2) primitives as Rust library crates that can be used by anyone
-   that wants to expand the protocol or implement a role (e.g. a pool wanting to support Sv2, a
-   mining-device/hashrate producer that wants to integrate Sv2 into their firmware, a Bitcoin node
-   that wants to implement Template Provider capabilities to build the `blocktemplate`).
+The goals of this project are to provide:
 
-1. The above Rust primitives as a C library, making it available to non-Rust users via a C
-   bindings.
+1. A robust set of Stratum V2 (Sv2) primitives as Rust library crates which anyone can use
```
Comment (Fi3) [inline README.md:47]: C library is correct

**Review by rrybarczyk — APPROVED** [review]

**Pull request merged**